game/scripts/vscripts/gamemode.lua

```diff
@@ -153,6 +155,7 @@ function GameMode:OnGameInProgress()
   InitModule(NGP)
   InitModule(HeroProgression)
   InitModule(SellBlackList)
+  InitModule(Courier)
```

Comment:
I think the `Courier` module is getting initialised too late for couriers to actually spawn since `OnGameInProgress` doesn't fire until minute 0 (a few seconds after heroes spawn).